app/src/pages/Devices/DeviceDetails/index.tsx

```diff
@@ -34,14 +34,17 @@ export function DeviceDetails(): JSX.Element | null {
         overflow={OVERFLOW_SCROLL}
         padding={SPACING_3}
       >
+        {/* TODO(va, 2022-06-17) update border color to 
+            COLORS.medGreyEnabled when PR #10664 is merged */}
         <Flex
           alignItems={ALIGN_CENTER}
           backgroundColor={C_WHITE}
```

Comment:
I think this would be a good time to remove the old const
```suggestion
          backgroundColor={COLORS.white}
```